### DISCUSSION TITLE
JUMPBUG

### DISCUSSION THREAD

**Discussion by hrna94** [danielkrupinski/Osiris#3695] [Ideas]
Hello everyone,

is it possible to add jumpbug to osiris?
Or can you let me know if there is a way?

Thanks

**Comment by Xsintashi** [comment]
https://github.com/danielkrupinski/Osiris/pull/1829 and some pasting knowledge

**Reply by hrna94** [reply]
Ive found it but actually I tried it but it is not working for me. Can you help me with it, pls?

**Comment by hrna94** [comment]
I've tried adding you but you are not accepting friend requests. You can
add me pls Spanky#6010
